s3restore_test.go

```diff
@@ -0,0 +1,155 @@
+package main_test
+
+import (
+	"os/exec"
+	"io/ioutil"
+	"bytes"
+	"net/http"
+	"time"
+	"regexp"
+
+    . "github.com/alphagov/paas-s3restore"
```

Comment:
> I believe vim-go is just a fancy wrapper around gofmt

That's underselling it massively. `gofmt` is just one of the things vim-go implements for you.

We should be `gofmt`ing all our Go code. I think that's worth fixing for consistency's sake. I'm going to spend a little time getting a linter hooked up so that we can cetch this sort of thing automatically.